packages/next/telemetry/trace/report/index.ts

```diff
@@ -27,14 +53,17 @@ if (process.env.TRACE_TARGET && !target) {
   )
 }
 
-export let reporter: Reporter
+let traceTargetReporter: Reporter
 
 if (target === TARGET.CONSOLE) {
-  reporter = reportToConsole
+  traceTargetReporter = reportToConsole
 } else if (target === TARGET.ZIPKIN) {
-  reporter = reportToZipkin
+  traceTargetReporter = reportToZipkin
 } else if (target === TARGET.JAEGER) {
-  reporter = reportToJaeger
+  traceTargetReporter = reportToJaeger
 } else {
-  reporter = reportToTelemetry
+  traceTargetReporter = reportToTelemetry
 }
+
+// JSON is always reported to allow for diagnostics
+export const reporter = new MultiReporter([reportToJson, traceTargetReporter])
```

Comment:
Should we put this behind an experimental flag initially?

Reply:
It is intentionally not behind an experimental flag to find any potential edge cases before a stable release.